types/artifacts.d.ts

```diff
@@ -122,6 +122,8 @@ declare global {
       Manifest: Artifacts.Manifest | null;
       /** The URL loaded with interception */
       MixedContent: {url: string};
+      /** Elements that obscure the Largest Contentful Paint element. */
+      ObscuringElements: Array<any>;
```

Comment:
I think either the name of this artifact needs to change (`ElementsObscurringLCPElement` ... mouthful), or we structure explicitly as `{lcp: <list of elements>}`

Reply:
I think I'm fine with a longer, specific name in this case actually. I just can't think of another use case for the artifact besides this specific audit and so I'm willing to deal with it.